src/utils/matter-parser.js

```diff
@@ -0,0 +1,98 @@
+const yaml = require('yaml');
+const fsp = require('fs').promises;
```

Comment:
That’s cool. I keep forgetting this is a thing and always wrap the functions with `require('util').promisify`. 😅

Reply:
Yeah, wrapping the `fs` api functions with `promisify` may be necessary if I would like to support the minimum maintained version, which is node v8.x which doesn't have that, but I'm fine with only supporting node v11.14.0 😀